src/app/examples/grid-angular.component.ts

```diff
@@ -18,6 +18,8 @@ import { CustomAngularComponentEditor } from './custom-angularComponentEditor';
 import { CustomAngularComponentFilter } from './custom-angularComponentFilter';
 import { CustomTitleFormatterComponent } from './custom-titleFormatter.component';
 import { FilterNgSelectComponent } from './filter-ng-select.component';
+import { DropDownService } from './dropdown.service';
```

Comment:
Your new Service doesn't seem to be part of the `providers` of the App Module, is that something you forgot to do? You don't seem to instantiate that Service but instead using it as a static Service, I'm not sure that is a good idea since you won't be able to use Dependency Injection in the future